Dockerfile

```diff
@@ -36,12 +12,9 @@ RUN pip install -e .[nlp] --no-use-pep517
 
 # ---- test -----
 # we need to install test dependencies before, so we cannot use 'base_baal' as base image
-FROM continuumio/miniconda3 as test_baal
+FROM pytorch/pytorch:1.8.0-cuda11.1-cudnn8-devel as test_baal
 
```

Comment:
do we need pytorch base for tests because it makes the image big?

Reply:
It is easier this way and it mimics our dockerfile which is good I think.

Reply:
okidou